### PR TITLE
added boardfiles for ruuvitag

### PR DESCRIPTION
**Pull request by pentabarf**

### CODE REVIEW

File: boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+* 6 = SWO
+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+nrfjprog --family nrf52 --eraseall
```
Comment (dbkinder) [inline boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst]: Where does the ``firmware.hex`` file come from?

This looks like command line steps for flashing.  If so, use a code block directive:

    .. code-block:: console

       nrfjprog --family nrf52 --eraseall
       nrfjprog --family nrf52 --program firmware.hex
       nrfjprog --family nrf52 --reset

File: boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+=========
+
+
+Testing the LEDs and buttons in the nRF52 DK
+********************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+.. code-block:: console
```
Comment (dbkinder) [inline boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst]: Replace this code block with a bullet list that points to the documentation for these samples:

    * :ref:`blinky-sample`
    * :ref:`button-sample`

The name given in the ``:ref:`` tag is the label at the top of the .rst file for that sample's documentation.

File: boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTT       | on-chip    | console              |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF52 DK website`_ and `Nordic Semiconductor Infocenter`_
```
Comment (dbkinder) [inline boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst]: You're missing the definitions of these two link targets.  At the bottom of this file add them with:

    .. _nrf52 DK website:  url to site
    .. _Nordic Semiconductor Infocenter: url to site

File: boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst [hunk 4]
```diff
@@ -0,0 +1,169 @@
+.. _nrf52_ruuvi:
+
+nRF52-ruuvi
+##############
+
+Overview
+********
+
+RuuviTag is a generic sensor node, that fits perfectly with many uses.
```
Comment (dbkinder) [inline boards/arm/nrf52_ruuvi/doc/nrf52_ruuvi.rst]: This is too much like marketing, how about replacing lines 9-11 with:

    RuuviTag is an advanced battery-operated open-source Bluetooth
    enabled sensor beacon platform capable of sending temperature, humidity, 
    pressure, and motion information.

**Review by dbkinder — CHANGES_REQUESTED** [review]
There are also problems with the commit message and sign off, see https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/20857/5/tests

**Review by dbkinder — APPROVED** [review]
+1 for doc changes, thanks.
There are still some CI build issues being reported though.

**Comment by ioannisg** [comment]
Hi, @pentabarf, to start with, would you be able to re-work the commit messages based on the Contribution guidelines? See, https://docs.zephyrproject.org/latest/contribute/contribute.html

**Comment by ioannisg** [comment]
@pentabarf  any plans to work on this PR?

**Comment by nashif** [comment]
no response for 3 months, closing.

**Pull request closed without merging**